aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -1131,6 +1133,9 @@ struct Brgemm : public KernelCache <BrgemmKey, GemmHelper> {
     } else if (dtype == ScalarType::BFloat16) {
       static bool bf16_support = dnnl::get_effective_cpu_isa() >= dnnl::cpu_isa::avx512_core;
       return bf16_support;
+    } else if (dtype == ScalarType::Byte || dtype == ScalarType::Char) {
+      static bool bit8_support = dnnl::get_effective_cpu_isa() >= dnnl::cpu_isa::avx512_core;
```

Comment:
Have we checked if int8 brgemm only need `avx512_core` instead of `avx512_vnni`?